packages/tradershub/src/features/cfd/components/MT5PlatformsList/MT5PlatformsList.tsx

```diff
@@ -48,14 +49,20 @@ const MT5PlatformsList = ({ onMT5PlatformListLoaded }: TMT5PlatformsListProps) =
                 </div>
             )}
             {isFetchedAfterMount &&
-                data?.map(account => {
-                    if (account.is_added)
-                        return <AddedMT5AccountsList account={account} key={`added-mt5-list-${account.loginid}`} />;
+                data?.map(MT5Account => {
+                    if (
+                        MT5Account.is_added &&
+                        MT5Account.is_virtual === activeTradingAccount?.is_virtual &&
```

Comment:
[Question]: How does this work for added MT5 real accounts?

Reply:
Let say we are testing for real account:  (_**MT5Account.is_added**_) = `True`   ,  (**_MT5Account.is_virtual_**) = `False` , (**_activeTradingAccount?.is_virtual_**) = `False`,  (**_MT5Account.account_type_** = `Real` ) , (**_accountType_** = `Real`)  => 

(True && (False = False=> True) && ( Real = Real => True) ) =>  True && True && True = True  , so It will render <AddedMT5AccountsList/> with added mt5 real accounts. 

hope it helps 😄